src/merkle_tree.rs

```diff
@@ -22,100 +24,116 @@ impl MerkleTree {
     pub fn new<T: std::convert::AsRef<[u8]>>(data: &[T]) -> MerkleTree {
```

Comment:
I think here its cleaner without the full path for AsRef, as its a common known trait

Reply:
I'll clean this up as well in the "final" version.

Thanks ^_^